Proposed changes creating Custom SIT in PowerShell (#2828)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/create-a-custom-sensitive-information-type-in-scc-powershell.md

- Cannot have an unbounded repeater (such as "\*" or "+") on a group.
    
  For example, "(xx)\*" and "(xx)+" will not pass validation.
  

> - Keywords have a maximum of 50 characters in Length.  If you have a keyword within a Group exceeding this, a suggested solution is to create the Group of terms as a [Keyword Dictionary](https://docs.microsoft.com/en-us/microsoft-365/compliance/create-a-keyword-dictionary) and reference the GUID of the Keyword Dictionary within the XML structure as part of the Entity for Match or idMatch in the file.

Reviewer: ```suggestion
- Keywords have a maximum of 50 characters in length.  If you have a keyword within a Group exceeding this, a suggested solution is to create the Group of terms as a [keyword dictionary](https://docs.microsoft.com/microsoft-365/compliance/create-a-keyword-dictionary) and reference the GUID of the keyword dictionary within the XML structure as part of the Entity for Match or idMatch in the file.
```

> - Each Custom Sensitive Information Type can have a maxium of 2048 keywords total.

Reviewer: ```suggestion
- Each custom sensitive information type can have a maximum of 2048 keywords in total.
```

> - When using the PowerShell Cmdlet there is a maximum return size of the Deserialized Data of approximately 1 megabyte.   This will affect the size of your XML file. Keep the uploaded file limited to a 512 megabyte maximum as a suggested limit for consistent results without error when processing.

Reviewer: ```suggestion
- When using the PowerShell cmdlet, there is a maximum return size of the deserialized data of approximately 1 megabyte.   This will affect the size of your XML file. Keep the uploaded file limited to a 512-megabyte maximum as a suggested limit for consistent results without error when processing.
```

> - The XML structure does not require formatting characters such as Spaces, Tabs or Carriage Return / Linefeed entries.  Take note of this when optimizing for space on uploads.

Reviewer: ```suggestion
- The XML structure does not require formatting characters such as spaces, tabs, or carriage return/linefeed entries.  Take note of this when optimizing for space on uploads.
```
